playwright/pages/FollowUpPage.js

```diff
@@ -0,0 +1,31 @@
+/* eslint-disable no-useless-constructor */
+/* eslint-disable prettier/prettier */
```

Comment:
Please remove all instances of this disable and format correctly.

Reply:
removed